packages/hooks/package.json

```diff
@@ -38,7 +38,7 @@
     "zxcvbn": "^4.4.2"
   },
   "peerDependencies": {
-    "@apollo/client": "^3.3.16",
+    "@apollo/client": "^3.7.10",
```

Comment:
this is gonna be interesting tho. I wonder if @GetJobber/frontend-foundations peeps are updating apollo in JO. If they are, then this should be fine. Otherwise, this might be a breaking change.

Reply:
Don't recall this needing to be upgraded in JO - can't see any FeF subtasks for it.  Still at 3.3.16 in JO.

Reply:
yeah, we're bumping this because it's only for R16,17 
![image](https://user-images.githubusercontent.com/15986172/222829166-d743f524-820b-4681-a2dd-3d391395e39f.png)

Reply:
Oooh okay, dang 😬

Reply:
also, based on the test that failed on this PR, it might break some shiiiiiii

Reply:
Thanks for pointing this out. [Testing right now](https://github.com/GetJobber/Jobber/pull/34139) to see what this means for JO. I do not understand why `npm ls` does not surface that up.

Reply:
actually just found this out after upgrading to node 18 on my react 18 branch. That's the first error I got 😆

Reply:
So weird. Damn you `npm` 👊

Reply:
graphql-ruby-client doesn't support this apollo client version in JO, so I got this [PR](https://github.com/GetJobber/Jobber/pull/34147) to bump it

Reply:
I got a PR open for this. I wonder if we need to be specific about the version or if we can/should do `>=3.7.10` or something. I also wonder why the peer Dependency is different than the "real" dependency.

Reply:
to answer your question there @timoteialbu I think it's mostly to make sure that what we use in Atlantis is the same as what we would use in other places since this whole gql thing is finicky... i think.

Reply:
Would we ever get rid of `useCollectionQuery` in favor of something apollo has built in so that we do not have this dependency to begin with? I suppose its not the worst dependency to have cause its used in one place and its easy enough to update atlantis, but still.

Reply:
that is a question that I do not have an answer to. IMO, we should but that's a question best answered by the people that uses this. Maybe ask in #engineering, #tech-mobile/web?